velox/docs/functions/spark/string.rst

```diff
@@ -53,6 +53,13 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT conv("11abc", 10, 10); -- '11'
         SELECT conv('H016F', 16, 10); -- '0'
 
+.. spark:function:: empty2null(input) -> varchar
```

Comment:
Curious if this function is more efficient then `if(input <> '', input, null)` expression.